Safewaan/HonkedIn
# Styling for Create Resource Page

## Trello ticket link

[Developer Task: Implement UI/UX feedback to the forms](https://trello.com/c/EVvyVCIw/125-developer-task-implement-ui-ux-feedback-to-the-forms )



## Implementation description
* Add Styling for the create resource page



## Steps to test
1. Verify a resource can be created
2. Verify error messages appear when required text fields are not given
